http-core/src/main/scala/org/apache/pekko/http/impl/util/StreamUtils.scala

```diff
@@ -260,15 +260,6 @@ private[http] object StreamUtils {
       }
   }
 
-  /**
-   * Similar idea than [[FlowOps.statefulMapConcat]] but for a simple map.
-   */
-  def statefulMap[T, U](functionConstructor: () => T => U): Flow[T, U, NotUsed] =
-    Flow[T].statefulMapConcat { () =>
```

Comment:
Can you deprecate this? We can't remove public methods without deprecating them first.

Reply:
The StreamUtil is private[http].

Reply:
![image](https://github.com/apache/incubator-pekko-http/assets/501740/26a50f7d-ff70-4c18-9d81-4509e4b77b69)